File: 001/001.rb
#!/usr/local/bin/ruby -w

# Find the sum of all the multiples of 3 or 5 below 1000.

def main
    total = 0
    (1..999).each do |n|
        total += n if (n % 3).zero? || (n % 5).zero?
    end
    puts total
end

main

File: 001-010/001/001.py
#!/usr/bin/env python

# Find the sum of all the multiples of 3 or 5 below 1000.


def main():
    total = 0
    for x in range(1, 1000):
        if x % 3 == 0 or x % 5 == 0:
            total = total + x
    print(total)


if __name__ == "__main__":
    main()
